src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/solvers/terminals/ChangeAllToSameColorSolver.kt

```diff
@@ -50,15 +51,35 @@ object ChangeAllToSameColorSolver {
         }
         GlStateManager.translate(0f, 0f, 299f)
         for ((slot, clicks) in mapping) {
-            val betterOpt = if (clicks.first > -clicks.second) clicks.second else clicks.first
+            var betterOpt = if (clicks.first > -clicks.second) "${clicks.second}" else "${clicks.first}"
+            var color = CommonColors.WHITE
+            if(Skytils.config.changeToSameColorMode == 1){
+                val leftClick = when(Minecraft.getMinecraft().gameSettings.keyBindAttack.keyCode){
```

Comment:
yeah i'm not really too sure about this one

Reply:
Im gonna remove it for now

Reply:
Im gonna remove it ig

Reply:
removed :cry: